types/warmup.d.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License.
+
+import { FunctionOptions, FunctionResult, FunctionTrigger, RetryOptions } from './index';
+import { InvocationContext } from './InvocationContext';
+
+export type WarmupHandler = (warmupContext: object, context: InvocationContext) => FunctionResult;
```

Comment:
What does `warmupContext` actually look like when you run the function?

Reply:
I am not able to glean much information from the runtime logs. All I can see is that it is an empty object with no prototype. I compared the types used by the other languages in the docs and they seem to be using a simple object as well for the type signature. I can't think of a way to gather more information at the moment. Do you have any tips?

Reply:
You may have done this already, but I would just do `context.log(JSON.stringify(warmupContext));`

Reply:
Yeah, I tried, but all it logs out is an empty object: `{}`.

Reply:
Okay I feel like `object` makes it seem like we just forgot the types. I'd rather do one of the following:
1. Add a placeholder/empty interface like [this one](https://github.com/Azure/azure-functions-nodejs-library/blob/v4.x/types/eventGrid.d.ts#L18)
2. Somehow change the WarmupHandler so that it doesn't pass the input and only passes the InvocationContext

Reply:
I agree. Just having `object` does not look right. I would like to do option two, but I see a `warmupContext` in the docs for all the other languages, so it must be there for a reason (I hope). 

I added an empty interface for now. I can keep digging to see if I find more information on what the interface should actually look like.

Reply:
> so it must be there for a reason (I hope).

I kinda doubt it. Every other trigger type has an input - my guess is they just did an empty object because it was easier than breaking the pattern.